src/TBuffer.h

```diff
@@ -95,7 +95,9 @@ struct TMxpElement
 
 class TBuffer
 {
-    static const QMap<QString, QVector<QChar>> csmEncodingTable; // private!
```

Comment:
You will need to add:

        Q_DECLARE_TR_FUNCTIONS(TBuffer) // Needed so we can use tr() even though TBuffer is NOT derived from QObject

here at the start of the class declaration, so you can use the `tr(`...`)` wrapper to mark (GUI) translatable text...!